### PR TITLE
get rid of warning

### PR DESCRIPTION
**Pull request by fengziyue**
I got this warning when training panoptic segmentation on S3DIS:

`points3d/torch_points3d/datasets/panoptic/utils.py:33: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor)`

So I modified that line.

### CODE REVIEW

**Review by nicolas-chaulet — APPROVED** [review]
Thanks for the patch!
